source/atomics.tex

```diff
@@ -1340,7 +1340,7 @@
 
 \pnum
 The following operations perform arithmetic computations.
-The key, operator, and computation correspondence are identified
+The key, operator, and computation correspondence is identified
```

Comment:
@jensmaurer: Instead of having these perpetual "guess the association" games, how about we say "The correspondence among key, operator, and computation is identified..."?

Reply:
Sure.